bundles/org.openhab.binding.tibber/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -90,4 +90,14 @@
 		</tags>
 		<state readOnly="true" pattern="%.0f %unit%"></state>
 	</channel-type>
+	<channel-type id="price-event">
+		<kind>trigger</kind>
+		<label>Price Event</label>
```

Comment:
Do you want to reserve the type for other kind of events as well? You could consider shortening type id to simply `event` and the label to "Event" (see also #16704).

Reply:
I switched to your notion, the event `DAY_AHEAD_AVAILABLE`  nails it.
Also I switched channel-type id to the generic notion.